lib/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ else()
   # executables (e.g. poclcc -> libOpenCL -> libLLVM)
   # and this must be specified on new systems,
   # otherwise linking fails.
-  set(OPENCL_LIBS "${PTHREAD_LIBRARY};${POCL_LIBRARY_NAME};${POCL_TRANSITIVE_LIBS}")
+  set(OPENCL_LIBS "${PTHREAD_LIBRARY};${POCL_LIBRARY_NAME};${POCL_TRANSITIVE_LIBS};${LLVM_LDFLAGS}")
```

Comment:
Non-ideal to put linker flags to `OPENCL_LIBS` that should only have libraries but it'll do for now.